### DISCUSSION TITLE
App feels heavy ?

### DISCUSSION THREAD

**Discussion by russssl** [toeverything/AFFiNE#6146] [General]
So I tried to use it and it is cool and all but the major issue with me is that the app feels "heavy", with long animation, and sometimes performance - it is slow. just not fast compared to Notion for example. I would love this problem to be addressed somehow

**Comment by russssl** [comment]
also connected to this problem, the app is too crowded, too many elements are displayed, too many options at the same screen

**Comment by EYHN** [comment]
Maybe try 0.13.0 ?  I feel there have been some performance improvements
